src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,65 @@
+package tests;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import util.Property;
+
+public class BaseTest {
+	public static WebDriver driver;
+	
+	protected static final int WAIT_ELEMENT_TIMEOUT = 5;
+	
+	public void waitForElementPresent(By locator) {
+		new WebDriverWait(driver, WAIT_ELEMENT_TIMEOUT).until(ExpectedConditions.presenceOfAllElementsLocatedBy(locator));
+	}
+	
+	public void waitForElementVisible(By locator) {
+		new WebDriverWait(driver, WAIT_ELEMENT_TIMEOUT).until(ExpectedConditions.visibilityOfElementLocated(locator));
+	}
+	
+	@BeforeClass
+	public void setUp() {
+		if(driver == null) {
```

Comment:
I think this is too straight forward. 
Refactor to give the ability to use 2 types of drivers -- Chrome and FF. Create Enum and WebDriverProvider aka Factory to provide needed driver